src/content/docs/reference/configuration.mdx

```diff
@@ -29,71 +31,74 @@ If you have problems with resolving the physical file, you can use the one publi
 
 ## `extends`
 
-A list of paths to other JSON files. Biome resolves and applies the options of the files contained in the `extends` list, and eventually applies the options contained in the `biome.json` file.
+A list of paths to other Biome configuration files. Biome resolves and applies
+the configuration settings from the files contained in the `extends` list, and
+eventually applies the options contained in this `biome.json`/`biome.jsonc`
+file.
 
-## `files`
-
-### `files.maxSize`
+Files are provided in order from least relevant to most relevant, with the
+settings in the current file being leading.
```

Comment:
This phrase is a bit out of context. What "least relevant to most relevant" mean? You probably wanted to talk about the **order** of the items inside `extends`, is that right? If so, than we should mention it and maybe provide an example.

Reply:
Yes indeed, I’ll try to reword this a bit.